### DISCUSSION TITLE
Help, red mark on custom items (3.3.5)

### DISCUSSION THREAD

**Discussion by Mellstroyez** [TrinityCore/TrinityCore#26489] [Q&A] [answered]
Hello, I need your help, the third day I can't figure out what I'm doing wrong.

Created a new item, added it to item_template in the database.
Edited item.dbc, added a new item there.
Created an empty mpq file, created a DBFilesClient folder there and put the modified item.dbc there.
The mpq file called "patch-x.mpq" was transferred to the Data folder of the game client, deleted the Cache folder, rebooted the server and entered the game. But the new item is still a red mark.

What am I doing wrong, please tell me.

**Comment by ghost** [comment]
You may be interested to know that most of the TrinityCore contributors are focused on the source and non-custom material.

I recommend using either the TC Forum pages (if you have got lots of time and patience) or the various Discord channels for more immediate replies.

**Comment by Aokromes** [comment, marked as answer]
> Hello, I need your help, the third day I can't figure out what I'm doing wrong.
> 
> Created a new item, added it to item_template in the database.
> Edited item.dbc, added a new item there.
> Created an empty mpq file, created a DBFilesClient folder there and put the modified item.dbc there.
> The mpq file called "patch-x.mpq" was transferred to the Data folder of the game client, deleted the Cache folder, rebooted the server and entered the game. But the new item is still a red mark.
> 
> What am I doing wrong, please tell me.

you are doing nothing incorrect, it's client limitation.
